src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -217,7 +217,7 @@ bool ContextAttribute::calculateOperator
       return false;
 
     default:
-      LM_E(("Runtime Error (unknown attribute type: %d)", valueType));
+      LM_E(("Runtime Error (unknown attribute type: %d on attribute %s)", valueType, name.c_str()));
```

Comment:
```suggestion
      LM_E(("Runtime Error (unknown attribute value type: %d on attribute %s)", valueType, name.c_str()));
```

To homogenize all these similar LM_E traces.